plugins/CoreAdminHome/OptOutManager.php

```diff
@@ -145,6 +199,16 @@ public function getOptOutView()
             ? $language
             : LanguagesManager::getLanguageCodeForCurrentUser();
 
+        $this->addQueryParameters(array(
+            'module' => 'CoreAdminHome',
+            'action' => 'optOut',
+            'language' => $lang,
+        ), false);
+
+        if ($this->isUserAgentSafari() && $trackVisits) {
```

Comment:
Just FYI in case someone has a look at it as well: This used to be dependent on `loadInNewWindow` in the view {% if loadInNewWindow %}&amp;setCookieInNewWindow=1{% endif %}` and the variable used to be set in the view. It's good that it is now done in the class I reckon.